client/cli/utils.go

```diff
@@ -0,0 +1,37 @@
+package cli
```

Comment:
Let's make sure to run `make license` to include the copyright header!

Reply:
[`cd24a3d` (#4)](https://github.com/noble-assets/authority/pull/4/commits/cd24a3d9e28383485b3ea38afd49dd96ab90e3a1)